org.eclipse.xtext.common.types.eclipse.tests/tests/org/eclipse/xtext/common/types/access/xtext/ui/JdtBasedConstructorScopeTest.java

```diff
@@ -85,7 +85,7 @@ public boolean apply(IEObjectDescription input) {
 		assertFalse(Iterables.any(contents, new Predicate<IEObjectDescription>() {
```

Comment:
should be assertTrue i thing. @szarnekow what do you think?

Reply:
Having debugged the test I can say that AbstractCollection and AbstractList are present, I think this should be assertTrue as well.